IPython/core/usage.py

```diff
@@ -76,6 +76,22 @@
 At your system command line, type 'ipython -h' to see the command line
 options available. This document only describes interactive features.
 
+GETTING HELP
+------------
+
+Within IPython you have various way to access help:
+
+  ?         -> Introduction and overview of IPython's features (this screen).
+
+  %quickref -> Quick reference of all IPython specif syntax and magics.
+
+  object?   -> Details about 'object', use 'object??' for extra details.
```

Comment:
Sounds good, I'd only adjust this:

On Fri, Aug 26, 2016 at 5:24 AM, Carol Willing <notifications@github.com>
wrote:

> object?? -> Provides more detailed, verbose information about 'object'


to simply "More detailed, ...". removing "provides": the others don't use
that verb, and it makes the line longer than 80 chars.

Thanks @willingc!


-- 
Fernando Perez (@fperez_org; http://fperez.org)
fperez.net-at-gmail: mailing lists only (I ignore this when swamped!)
fernando.perez-at-berkeley: contact me here for any direct mail